src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -2904,7 +2915,7 @@ def create_private_endpoint(cmd, resource_group_name, private_endpoint_name, sub
     else:
         private_endpoint.private_link_service_connections = [pls_connection]
 
-    return client.create_or_update(resource_group_name, private_endpoint_name, private_endpoint)
+    return client.begin_create_or_update(resource_group_name, private_endpoint_name, private_endpoint)
```

Comment:
Do we need support no_wait for this long running operation?

Reply:
Yes, usually LRO need support no_wait, but actually some of our code doesn't follow this guideline. 
In this PR, we just focus on migration and keep the same behavior as before.
For improvement, we can do it in future PRs.